.buildkite/react-native-pipeline.full.yml

```diff
@@ -53,12 +55,12 @@ steps:
         key: "build-react-native-ios-fixture-old-arch-full"
         timeout_in_minutes: 30
         agents:
-          queue: "macos-12-arm"
+          queue: "macos-14"
         env:
           NODE_VERSION: "18"
           RN_VERSION: "{{matrix}}"
           BUILD_IOS: "true"
-          DEVELOPER_DIR: "/Applications/Xcode14.app"
+          XCODE_VERSION: "15.3.0"
```

Comment:
This no longer needs the full path?

Reply:
nope! the new build servers have moved over to using `XCODE_VERSION` instead of `DEVELOPER_DIR` - which is handy because you don't need to know where it's installed

Reply:
sublime! 👌🏻